src/containers/Workspace.js

```diff
@@ -45,4 +40,4 @@ function mapDispatchToProps(dispatch) {
 export default connect(
   mapStateToProps,
   mapDispatchToProps,
-)(Workspace);
+)(resizableFlex(2)(Workspace));
```

Comment:
This is very clear - allows for 2 resizable containers in the workspace component.

Reply:
@pwjablonski good question. The basic situation here is that `resizableFlex` is a function that returns a function that returns a component. So, calling `resizableFlex(2)` means “make me a function that takes a component, and decorates it with a controller for a size 2 resizable flex container, returning the decorated component”. Then calling that function with the `Workspace` component as an argument gives us the actual decorated component.

Does that clarify?

Also occurs to me that this PR really needs some tests. In particular I am wondering if the inner workings of the `resizableFlex` module may be a bit easier to understand with the aid of some unit tests. Of course I should also add tests for the new reducer.